forge.config.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+  packagerConfig: {
+    osxSign: {
+      identity: 'Developer ID Application: MyCrypto Inc (43RN6VJTB7)',
+      'hardened-runtime': true,
+      'gatekeeper-assess': false,
```

Comment:
Why is this disabled?

Reply:
This also needs to be disable since we use self signed certificates